syne_tune/optimizer/schedulers/searchers/searcher.py

```diff
@@ -519,66 +548,90 @@ def get_config(self, **kwargs):
         if new_config is not None:
             # Add config to exclusion list so that it won't be suggested again
             self._excl_list.add(new_config)
-            if self._debug_log is not None:
-                self._debug_log.set_final_config(new_config)
-                # All get_config debug log info is only written here
-                self._debug_log.write_block()
+
+            # Write debug log for the the config
+            trial_id = kwargs.get("trial_id")
```

Comment:
To be consistent with other searcher's get_config(), I didn't make it an argument. I use a different format of log if the trial_id is not specified to prevent error.